packages/terra-form-input/tests/jest/InputField.test.jsx

```diff
@@ -21,6 +21,14 @@ describe('InputField', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should render placeholder when showMpageFusionPlaceholder is passed', () => {
+    const textarea = <InputField inputId="test-input" label="label" inputAttrs={{ placeholder: 'Enter The Text' }} />;
+    const wrapper = shallow(textarea);
+    const placeholderInput = (wrapper.find('Field')).find('Input#test-input');
+    expect(placeholderInput.props()).toHaveProperty('placeholder', 'Enter The Text');
```

Comment:
as per the code `<InputField inputId="test-input" label="label" inputAttrs={{ placeholder: 'Enter The Text' }} />` placeholder should be null here, since `showMpageFusionPlaceholder` prop is not passed placeholder value should be null in this case